core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -5,18 +5,13 @@
       <div class="col-lg-12 col-md-12 col-sm-12">
         <div ng-if="editabilityService.isEditable()">
           <div role="form" class="form-horizontal">
-            <div class="form-group" ng-class="{'has-error': !explorationTitleService.displayed}">
-              <label for="explorationTitle" class="col-lg-2 col-md-2 col-sm-2">Title</label>
-              <div class="col-lg-10 col-md-10 col-sm-10">
-                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" ng-trim="false">
-                <span class="help-block" style="font-size: smaller">
-                  <em>Please use at most 40 characters, so that this fits in the summary card.</em>
-                  <span ng-if="explorationTitleService.displayed.length > 0">
-                    <em>Characters used: <[(explorationTitleService.displayed.length)]>/40</em>
-                  </span>
-                </span>
-              </div>
-            </div>
+            <title-field
+              title="Title"
```

Comment:
Follow existing style: move this to previous line, left-align other attrs on subsequent lines with 'title'. Same thing for the one in exploration_metadata_modal_directive.html.